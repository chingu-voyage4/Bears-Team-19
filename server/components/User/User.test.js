const request = require('supertest');
const app = require('../app.js');

describe('Auth Routes', () => {
  test('Authentication routes return false with get request', async () => {
    const response = await request(app).get('/auth/register');
    expect(response.statusCode).toBe(404);
  });

  test('Auth Route returns response with post request', async () => {
    const object = {
      success: false,
    };

    const response = await request(app).post('/auth/register');
    expect(response.statusCode).toBe(200);
    expect(response.body).toMatchObject(object);
    expect(response.body).toMatchSnapshot();
  });
});
