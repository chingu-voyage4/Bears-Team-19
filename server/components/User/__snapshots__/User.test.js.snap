// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth Routes Auth Route returns response with post request 1`] = `
Object {
  "success": false,
}
`;
